editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1539,6 +1545,36 @@ export const MetadataUtils = {
       return this.findNearestAncestorFlexDirectionChange(elementMap, parentPath)
     }
   },
+  isFocusableComponent(
+    path: TemplatePath,
+    components: UtopiaJSXComponent[],
+    metadata: ElementInstanceMetadataMap,
+    imports: Imports,
+  ): boolean {
+    const elementName = MetadataUtils.getJSXElementName(path, components, metadata)
+    const elementInstancePath = TP.isInstancePath(path)
+      ? path
+      : TP.instancePathForElementAtScenePath(path)
+    const element = MetadataUtils.getElementByInstancePathMaybe(metadata, elementInstancePath)
```

Comment:
We have a function for doing exactly this `findElementByTemplatePath`, which exists for smoothing the transition between the separate path types and a future where they don't exist

Reply:
fixed!